docs/examples/linkedin.rst

```diff
@@ -9,31 +9,39 @@ command line interactive example below.
 
 .. code-block:: pycon
 
+    >>> # Imports
+    >>> import os
+    >>> from requests_oauthlib import OAuth2Session
+
+    >>> # Set environment variables
+    >>> os.environ['OAUTHLIB_INSECURE_TRANSPORT'] = '1'
+
     >>> # Credentials you get from registering a new application
     >>> client_id = '<the id you get from linkedin>'
     >>> client_secret = '<the secret you get from linkedin>'
 
-    >>> # OAuth endpoints given in the LinkedIn API documentation
-    >>> authorization_base_url = 'https://www.linkedin.com/uas/oauth2/authorization'
-    >>> token_url = 'https://www.linkedin.com/uas/oauth2/accessToken'
+    >>> # LinkedIn OAuth2 requests require scope and redirect_url parameters.
+    >>> # Ensure these values match the auth values in your LinkedIn App 
+    >>> # (see auth tab on LinkedIn Developer page)
+    >>> scope = ['r_liteprofile']
+    >>> redirect_url = 'http://127.0.0.1'
 
-    >>> from requests_oauthlib import OAuth2Session
-    >>> from requests_oauthlib.compliance_fixes import linkedin_compliance_fix
+    >>> # OAuth endpoints given in the LinkedIn API documentation
+    >>> authorization_base_url = 'https://www.linkedin.com/oauth/v2/authorization'
+    >>> token_url = 'https://www.linkedin.com/oauth/v2/accessToken'
 
-    >>> linkedin = OAuth2Session(client_id, redirect_uri='http://127.0.0.1')
-    >>> linkedin = linkedin_compliance_fix(linkedin)
+    >>> linkedin = OAuth2Session(client_id, redirect_uri='http://127.0.0.1', scope=scope)
 
     >>> # Redirect user to LinkedIn for authorization
     >>> authorization_url, state = linkedin.authorization_url(authorization_base_url)
-    >>> print 'Please go here and authorize,', authorization_url
+    >>> print("Please go here and authorize, {}".format(authorization_url))
 
     >>> # Get the authorization verifier code from the callback url
-    >>> redirect_response = raw_input('Paste the full redirect URL here:')
+    >>> redirect_response = input('Paste the full redirect URL here:')
 
     >>> # Fetch the access token
-    >>> linkedin.fetch_token(token_url, client_secret=client_secret,
-    ...                      authorization_response=redirect_response)
+    >>> linkedin.fetch_token(token_url,client_secret=client_secret,include_client_id=True,authorization_response=redirect_response)
```

Comment:
Can you break this across multiple lines? It's hard to read all on one line like this.

Reply:
No problem. This file as a whole is weird to me. Not sure why it was made into an .rst, and even if it's something the OC wanted documented this is sort of a pain to edit. Perhaps as a back log item these can be reformatted so that they render well in documentation and are more user friendly e.g. pull down and hit play.